<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winMainClass</class>
 <widget class="QMainWindow" name="winMainClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tic-Tac-Toe++ Alpha</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QGridLayout" name="grdGame">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="lblTurn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:blue;
color:white;
font-weight:bold;
font-size:24px;</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpDificil">
        <property name="minimumSize">
         <size>
          <width>199</width>
          <height>116</height>
         </size>
        </property>
        <property name="title">
         <string>Difficulty</string>
        </property>
        <widget class="QRadioButton" name="radEasy">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Easy</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radNormal">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Normal</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radHard">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Hard</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radImp">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:red;font-weight:bold;</string>
         </property>
         <property name="text">
          <string>Impossible</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpOpp">
        <property name="minimumSize">
         <size>
          <width>199</width>
          <height>60</height>
         </size>
        </property>
        <property name="title">
         <string>Opponent</string>
        </property>
        <widget class="QRadioButton" name="radComputer">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Computer</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radHuman">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Human</string>
         </property>
        </widget>
        <zorder>radHuman</zorder>
        <zorder>radComputer</zorder>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTaunt">
        <property name="maximumSize">
         <size>
          <width>285</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layBtns">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnNew">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;New Game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExit">
          <property name="text">
           <string>E&amp;xit Game</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mnuMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="mnuNewGame"/>
    <addaction name="mnuExit"/>
   </widget>
   <widget class="QMenu" name="mnuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="mnuAbout"/>
    <addaction name="separator"/>
    <addaction name="mnuAboutQt"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="mnuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mnuNewGame">
   <property name="text">
    <string>&amp;New Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="mnuExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="mnuAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="mnuAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="winmain.qrc"/>
 </resources>
 <connections/>
</ui>
